test.sh

```diff
@@ -212,6 +212,8 @@ STAGE="lints"
 if [[ "${RUN[@]}" =~ "$STAGE" ]] ; then
   print_heading "Running Lints"
   golangci-lint run --timeout 9m ./...
+  # Implicitly loads staticcheck.conf from the root of the boulder repository
```

Comment:
Can it be given a path to a config file, so that the config file can be a hidden file (like .golangci.yml or .codespell.ignore.txt)?